nightguard/DateExtension.swift

```diff
@@ -52,41 +52,25 @@ extension Date {
         
         return dateFormatter.string(from: self)
     }
+
     func toUTCMillis() -> String {
         
-        return String(Int64((self.toGlobalTime().timeIntervalSince1970 * 1000.0).rounded()))
+        return String(Int64((self.timeIntervalSince1970 * 1000.0).rounded()))
     }
 
     func convertToIsoDate() -> String {
         
-        let dateFormatter = DateFormatter()
-        dateFormatter.dateFormat = "yyyy-MM-dd"
-        dateFormatter.locale = Locale(identifier: "en_US")
-        dateFormatter.timeZone = TimeZone.init(secondsFromGMT: 0)
-        
-        return dateFormatter.string(from: self)
+        return ISO8601DateFormatter.string(
+            from: self,
+            timeZone: TimeZone.gmt,
```

Comment:
this also needs 16 but can easily be changed to `TimeZone.init(identifier: "UTC")!`

Reply:
Yes - would be good. It's always a big deal to loose people with older hardware ;) .